docs/_docs/reference/metaprogramming/macros-spec.md

```diff
@@ -144,7 +144,7 @@ environment of the stack.
 
                Es |- t1: T2 -> T    Es |- t2: T2
                ---------------------------------
-                      Es |- t1 t2: T
+                      Es |- t1 t2 : T
```

Comment:
This change is not consistent with what's around. It should be reverted.

Reply:
```suggestion
                      Es |- t1 t2: T
```